cmake/ProjectCompilerPostConfig.cmake

```diff
@@ -1,115 +1,15 @@
 IF (${Trilinos_ENABLE_Kokkos})
```

Comment:
This needs to be:

```
IF (${PROJECT_NAME}_ENABLE_Kokkos)
```

Reply:
Thanks @bartlettroscoe , I will update the PR with this change.

Reply:
@bartlettroscoe this change was addressed in commit 050fe00155cb4186773cd3a3cd1835a3af89af35

Reply:
@bartlettroscoe I clicked the re-request review button, I didn't want to dismiss your previous review for this requested change.